hazelcast/include/hazelcast/client/protocol/ClientMessage.h

```diff
@@ -600,7 +600,7 @@ class HAZELCAST_API ClientMessage
                                typename T::value_type::second_type>,
                      typename T::value_type>::value &&
         hazelcast::util::is_trivial_or_uuid<typename T::value_type::first_type>::value &&
-        std::is_trivial<typename T::value_type::second_type>::value,
+        hazelcast::util::is_trivial_or_uuid<typename T::value_type::second_type>::value,
```

Comment:
I didn't touch this one or the one after because as far as I could see nobody formed `vector<pair<trivial, uuid>>` but no objection to the change.